src/igvfd/audit/compare_dict_values.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
```

Comment:
defyining class for function is questionable

Reply:
eliminated the class